moshi/src/test/java/com/squareup/moshi/JsonReaderTest.java

```diff
@@ -165,6 +165,20 @@ JsonReader newReader(String json) throws IOException {
     assertThat(reader.peek()).isEqualTo(JsonReader.Token.END_DOCUMENT);
   }
 
+  @Test public void failOnTrailingComma() throws IOException {
```

Comment:
we already have this, com.squareup.moshi.JsonUtf8ReaderTest#strictExtraCommasInMaps

Reply:
Ah! Thanks for pointing this out. I did not find it. Here are a few follow-up questions:

1. Should we assert for the error message like I did in my test?
2. Why is your test named `InMaps`?
3. Why is my test failing for `[Value]` or why is your test in `JsonUtf8ReaderTest` instead in `JsonReaderTest`?
4. Do you have a opinion about the `TrailingCommaException` proposal or do you prefer to discuss this in a separate issue?